CMakeLists.txt

```diff
@@ -252,7 +252,7 @@ OCV_OPTION(WITH_EIGEN "Include Eigen2/Eigen3 support" (NOT CV_DISABLE_OPTIMIZATI
   VISIBLE_IF NOT WINRT AND NOT CMAKE_CROSSCOMPILING
   VERIFY HAVE_EIGEN)
 OCV_OPTION(WITH_FFMPEG "Include FFMPEG support" ON
-  VISIBLE_IF NOT ANDROID AND NOT IOS AND NOT WINRT
+  VISIBLE_IF NOT IOS AND NOT WINRT
   VERIFY HAVE_FFMPEG)
```

Comment:
ON => NOT ANDROID (disabled by default on Android).
Use WITH_FFMPEG=ON in build scripts.

```
OCV_OPTION(WITH_FFMPEG "Include FFMPEG support" NOT ANDROID
  VISIBLE_IF NOT IOS AND NOT WINRT
  VERIFY HAVE_FFMPEG)
```